composer.json

```diff
@@ -20,12 +20,14 @@
         "symfony/finder": ">=2.0",
         "symfony/yaml": ">=2.0",
         "kriswallsmith/assetic": ">= 1.0",
-        "symfony/framework-bundle": ">=2.0.9"
+        "symfony/translation": ">=2.0.9",
+        "symfony/framework-bundle": ">=2.0.9",
+        "symfony/templating": ">=2.0.9"
```

Comment:
`symfony/templating` and `symfony/translation` are no longer implicitly required by `symfony/framework-bundle` in newer versions, which is why they are now explicit dependencies.